AppLovinAdapter/build.gradle.kts

```diff
@@ -70,10 +70,10 @@ dependencies {
 
     // For external usage, please use the following production dependency.
     // You may choose a different release version.
-    "remoteImplementation"("com.chartboost:chartboost-mediation-sdk:4.0.0")
+    "remoteImplementation"("com.chartboost:chartboost-mediation-sdk:12.4.1")
```

Comment:
👀 wait, looks like `4.0.0` was regex matched with this one?

Reply:
yeah let's use https://github.com/ChartBoost/chartboost-mediation-android-adapter-applovin/pull/70 which contains the correct version bump